### DISCUSSION TITLE
where to get database table schema file (sql) for github?

### DISCUSSION THREAD

**Discussion by anlexN** [community/community#62282] [Other Feature Feedback, Questions, & Ideas] [answered]
I have got [graphql schema file](https://docs.github.com/en/graphql/overview/public-schema), and [openapi schema file](https://github.com/github/rest-api-description) for github.

I am learning api and database design now. those two files help me a lots. but I need to learn github how to do your platform's database table design too.

by the way, github use mysql as database? why use postgresql?

**Comment by tuves** [comment]
Welcome to the GitHub Community, @anlexN, we're happy you're here! You are more likely to get a useful response if you are posting your question(s) in the applicable category and are explicit about what your project entails--giving a few more details might help someone give you a nudge in the right direction. I've gone ahead and moved it for you. Good luck!

**Reply by anlexN** [reply]
I have explained my purpose. is it ok?

**Reply by anlexN** [reply]
@tuves , do you forget me? please reply me......

**Reply by tuves** [reply]
That's fine @anlexN, I simply moved your question into the correct category, as the [Discover category](https://github.com/orgs/community/discussions/categories/discover) wasn't the right place for a general question. Good luck!

**Reply by anlexN** [reply]
who can I invite to answer my questions? I am very crazy. so long time to not solve my questions. oh my god!!!!!!

**Reply by Imen-Frigui** [reply]
https://github.blog/2018-06-20-mysql-high-availability-at-github/

**Reply by anlexN** [reply]
Teacher, you disunderstand my meaning. I need your database schema file (like table, function, and so on) behind github, not "High Availability" explaination.
like this
![image](https://github.com/community/community/assets/8686739/c4539c0d-a1e5-4ed4-935c-869fb644a353)

I need `sql` file or say `table schema` file, or say `table definition` file.

do you understand me now? please.

**Reply by Imen-Frigui** [reply]
https://livablesoftware.com/gitana-sql-git-repository-inspector/

**Reply by anlexN** [reply]
no, I don't need this. 
I need github.com itself database schema file.
do you understand me now? please.
Or can you recommand me to look for who can solve my problem?
@Imen-Frigui

**Reply by samus-aran** [reply, marked as answer]
Hi @anlexN 

The database schema for GitHub platform is not publicly available. I would suggest you seek other database schemas to help you learn database design. 🙂
